packages/api/tests/lib/rules/test-createRuleResources.js

```diff
@@ -352,3 +355,50 @@ test.serial('Creating an enabled SNS rule creates an event source mapping', asyn
     await awsServices.sns().deleteTopic({ TopicArn }).promise();
   });
 });
+
+
+test('buildPayload builds a lambda payload from the rule', async (t) => {
+  const ruleParams = {
+    type: 'onetime',
+    workflow,
+    meta: {
+      visibilityTimeout: 100,
+      retries: 4,
+    },
+    cumulus_meta: {
+      execution_name: 'fakeName',
+    },
+    payload: {
+      input: 'test',
+    },
+    execution_name_prefix: randomString(),
+    asyncOperationId: 1,
+  };
+  const rule = fakeRuleRecordFactory(ruleParams);
+  const expectedPayload = {
+    provider: rule.provider,
+    collection: rule.collection,
```

Comment:
I think this test is passing because `rule.provider` and `rule.collection` are undefined on the `payload` as well. So the test passes but maybe doesn't prove what we want?

Reply:
True [`bb74e3a` (#2871)](https://github.com/nasa/cumulus/pull/2871/commits/bb74e3aa1d4642235116c170a15beaa6bb63eb93#diff-f5c5b96c6230da09644104acbdca2268e22d32509bc5b275a5b4648a8939584eR363)